core/azure-core/src/main/java/com/azure/core/util/logging/ClientLogger.java

```diff
@@ -189,27 +206,14 @@ private void performLogging(String format, Object... args) {
         }
     }
 
-    /*
-     * Helper method to set the logging level.
-     * @param level Logging level
-     * @return Updated ClientLogger if the level is enabled.
-     */
-    private ClientLogger asLevel(int level) {
-        if (canLogAtLevel(level)) {
-            this.level = level;
-        }
-
-        return this;
-    }
-
-    /*
+    /**
```

Comment:
Last I've heard we should be using a multi-line comment instead of a JavaDoc for package private and private methods.

Reply:
Oh, I was not aware of that. I will revert these changes.